README.md

```diff
@@ -10,6 +9,52 @@
   </a>
 </div>
 
+<br>
+<div align="center">
+    <img 
+        src="https://img.shields.io/github/license/Avaiga/taipy?style=plastic&color=ff371a&labelColor=1f1f1f" 
+        alt="GitHub License" 
+        height="20px" 
+    />
+    &nbsp;
+    <img 
+        src="https://img.shields.io/badge/python%20version-3.8-ff371a?style=plastic&labelColor=1f1f1f" 
+        alt="Python version needed: 3.8" 
+    />
+    &nbsp;
+    <a target="_blank" href="https://github.com/Avaiga/taipy/releases">
+        <img 
+            alt="GitHub Release" 
+            height="20px" 
+            src="https://img.shields.io/github/v/release/Avaiga/taipy?display_name=release&style=plastic&color=ff371a&labelColor=1f1f1f"
+        ></a>
+</div>
+<br>
+
+<div align="center">
+    <a target="_blank" href="https://docs.taipy.io/en/latest/">
+        <img 
+            src="https://img.shields.io/badge/docs-ff371a?style=plastic&labelColor=1f1f1f&label=Explore" 
+            height="20px" 
+            alt="Explore the docs" 
+        ></a>
+    &nbsp;
+    <a target="_blank" href="https://docs.taipy.io/en/latest/gallery/">
+        <img 
+            src="https://img.shields.io/badge/examples-ff371a?style=plastic&labelColor=1f1f1f&label=Check" 
+            height="20px" 
+            alt="Demos & Examples" 
+        ></a>
```

Comment:
Here is the result. 
![image](https://github.com/user-attachments/assets/6188f606-e2df-4168-b147-a118d8be27b7)

while the badge is for exploring our gallery. It s not a check.

Reply:
okay I will update this badge to "Explore | Gallery"